/pose_transformer:
  ros__parameters:
    initial_orientation:
    - 0.0
    - 12.0
    - 90.0
    initial_position:
    - 0.4
    - -0.5
    - 0.0
    odom_in: /mins/imu/odom
    odom_out: /rover/odom
    path_in: /mins/imu/path
    path_out: /rover/path
    pose_in: /mins/imu/pose
    pose_out: /rover/pose
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
    yaw: 173.0

